service_identity/src/main/kotlin/org/thechance/service_identity/api/model/request/UpdateUserRequest.kt

```diff
@@ -0,0 +1,11 @@
+package org.thechance.service_identity.api.model.request
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class UpdateUserRequest(
```

Comment:
I think `UpdateUserRequest` is giving more as a function name more than a class name.